PhysicsTools/PatAlgos/python/slimming/slimming_cff.py

```diff
@@ -78,6 +78,7 @@
 (pp_on_AA_2018 | pp_on_PbPb_run3).toReplaceWith(
     slimmingTask, 
     cms.Task(slimmingTask.copy(), packedCandidateMuonID, packedPFCandidateTrackChi2, lostTrackChi2, centralityBin, hiHFfilters))
+pp_on_AA_2018.toReplaceWith(slimmingTask,cms.Task(primaryVertexAssociationCleaned,slimmingTask.copy()))
```

Comment:
Maybe I'm lost... but shouldn't this be replaced for `run2_miniAOD_pp_on_AA_103X`?